snap7/client/__init__.py

```diff
@@ -1239,7 +1245,7 @@ def as_tm_write(self, start: int, amount: int, data: bytearray) -> int:
         check_error(result)
         return result
 
-    def as_upload(self, block_num: int, _buffer, size) -> int:
+    def as_upload(self, block_num: int, _buffer: "_SimpleCData[Any]", size: "_SimpleCData[Any]") -> int:
```

Comment:
why do _all_ this type need to be a string?

Reply:
https://github.com/gijzelaerr/python-snap7/pull/516#discussion_r1623561951